pkg/client/rs/resource_server.go

```diff
@@ -13,6 +13,7 @@ import (
 
 type ResourceServer interface {
 	IntrospectionURL() string
+	TokenEndpoint() string
```

Comment:
Note: breaking change

Reply:
This again didn't seem to me as interface intended for public implementation
Correct me if I'm wrong